app/src/main/cpp/ControllerDelegate.h

```diff
@@ -13,6 +13,9 @@
 #include "Device.h"
 #include "GestureDelegate.h"
 
+#include "GLES/gl.h"
```

Comment:
Seems unfortunate to have this here, I think another proof that we need to get all this things out of Controller* classes.

Reply:
Yes, we have been abusing ControllerDelegate for a while. I already had plans for fixing that.